client/blocks/edit-gravatar/style.scss

```diff
@@ -0,0 +1,23 @@
+.edit-gravatar__image-container {
+	display: inline-block;
+	position: relative;
+	line-height: 0;
+
+	&.is-uploading::after {
+		content: '';
+		position: absolute;
+			top: 0;
+			right: 0;
+			bottom: 0;
+			left: 0;
+		background-color: rgba( $white, 0.75 );
+	}
+}
+
+.edit-gravatar__spinner {
+	position: absolute;
+		top: 50%;
+		left: 50%;
+	transform: translate(-50%, -50%);
+	z-index: z-index( 'root', '.edit-gravatar__spinner' );
```

Comment:
Taking a closer look, I don't believe we need the z-index here. The natural stacking order should put the spinner above the image.